Automattic/jetpack
# SSO: switch from module codebase to Connection package, part 3

Fixes https://github.com/Automattic/vulcan/issues/274
Fixes https://github.com/Automattic/vulcan/issues/254

## Proposed changes:

This PR is the final part of moving the SSO feature codebase to the Connection package.

It follows the work done in #36587, #36924, and #36989.

- We switch to loading the feature from the connection package in all parts of the Jetpack plugin codebase.
- We deprecate all module codebase classes and methods

### Other information:


## Jetpack product discussion

https://github.com/Automattic/vulcan/issues/251

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Jetpack > Settings > Security and enable the SSO feature.
* Log out and log back in, with your main user and with new users.
* Check your error logs and ensure that no deprecation notices appear in the logs.
* Check that there are no errors in CI.
